integreat_cms/cms/utils/pdf_utils.py

```diff
@@ -56,7 +56,7 @@ def generate_pdf(region, language_slug, pages):
     for page in pages:
         # add translation id and last_updated to hash key list if they exist
         page_translation = page.get_public_translation(language_slug)
-        if page_translation:
+        if page_translation and not page.archived:
```

Comment:
I have a few concerns about performance, if a page has very many archived children this causes a lot of additional database queries, but let's merge it first and then see whether it is actually a problem.

Reply:
Did not realize this, sorry. Do you have an alternative suggestion?

Reply:
An alternative solution would be to call `.cache_tree(archived=False)` on the queryset - but then it's not a queryset anymore, but a list, and we would have to perform the other calculations like `aggregate` in python... so a bit more effort, not sure whether it's justified, so let's try the simple approach first.